### PR TITLE
Add tutorial for expanding a MinIO Tenant

### PR DESCRIPTION
**Pull request by ravindk89**
# Summary

Tutorial on expanding MinIO Tenants in vSphere 7 (VCF 4.2)

# Testing

1. Clone the repository and checkout the branch. 
2. Run `npm install` . You may need to install a newer version of node (try v14.5+)
3. Run `pip install -r requirements.txt` (Requires Python 3). Recommend using a `venv` to keep your local Python clean.
4. Run `make html`
5. Run `python -m http.server --directory build/vsphere-monitoring/html/` - requires Python 3.7+ it seems.

### CODE REVIEW

**Comment by cesnietor** [comment]
which URI should at look at?
I'm getting this error when going to `http://localhost:8000/`
```
python3 -m http.server --directory build/vsphere-monitoring/html
Serving HTTP on 0.0.0.0 port 8000 (http://0.0.0.0:8000/) ...
127.0.0.1 - - [08/Feb/2021 13:12:16] code 404, message File not
```

**Review by kerneltime — APPROVED** [review]

**Comment by ravindk89** [comment]
@cesnietor I missed a trailing slash in my directions :( the contents are inside `/html/`

File: source/tutorials/manage-minio-tenant.rst [hunk 1]
```diff
@@ -12,11 +12,132 @@ Overview
...
+
+All zones in a MinIO tenant use the same storage policy. Ensure each ESXi
+host's locally attached drives are included in the storage policy used by the
+MinIO tenant.
+
+1) Open the :guilabel:`Add New Zone` Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the vSphere interface, select the cluster in which you want to 
+deploy the MinIO tenant.
```
Comment (cesnietor) [inline source/tutorials/manage-minio-tenant.rst]: Was this probably copy pasted? it seems that it is for creation of a tenant.

Comment (ravindk89) [reply source/tutorials/manage-minio-tenant.rst]: Ah, yes. Good catch, will fix this along with any other missteps you happen to find.

Comment (cesnietor) [reply source/tutorials/manage-minio-tenant.rst]: that's all I could see, everything else LGTM

**Pull request merged**